demonstrations/tutorial_qdet_embedding.py

```diff
@@ -0,0 +1,280 @@
+r"""Quantum Defect Embedding Theory (QDET)
+=========================================
+Performing efficient simulations of advanced materials and molecules remains a significant
+challenge in quantum chemistry and condensed matter physics due to the prohibitive costs of
+available methods. However, many interesting problems in quantum chemistry and condensed matter physics
+feature a strongly correlated region, which requires accurate quantum treatment, embedded within a
+larger environment that could be properly treated with cheaper approximations.  For example,
+this is the case for point defects in materials [#Galli]_, active site of catalysts [#SJRLee]_, surface phenomenon such
+as adsorption [#Gagliardi]_ and many more. Embedding theories serve as powerful tools for effectively
+addressing such problems by capturing the strong electronic correlations in the active region with high accuracy
+while accounting for the environment in a more approximate manner.
+
+The core idea behind embedding methods is to partition the system and treat the strongly correlated
+subsystem accurately, using high-level quantum mechanical methods, while approximating the effects
+of the surrounding environment in a way that retains computational efficiency. In this demo, we show
+how to implement the quantum defect embedding theory (QDET). The method has been successfully
+applied to study defects in CaO [#Galli]_ and to calculate excitations of the negatively charged NV center in diamond [#Galli2]_.
+An important advantage of QDET is its compatibility with quantum
+algorithms as we explain in the following sections. The method can be implemented for calculating
```

Comment:
```suggestion
algorithms as we explain in the following sections. It can be implemented for calculating
```
Trying to avoid repeating "method".